tools/Get-NuGetTool.ps1

```diff
@@ -0,0 +1,15 @@
+<#
```

Comment:
@AArnott I understand the hesitation, however, in this case, the 3rd party dependency would be a one time download, that is bootstrapped by a PowerShell script of all things :-) The installation is actually just the extraction of a nuget package into a local tools folder.